nibe/data/extensions.json

```diff
@@ -29,6 +29,15 @@
         "mappings": {
           "0": "Off"
         }
+      },
+      "47340": {
+        "factor": 1,
+        "max": 18.0,
+        "mappings": {
+          "0": "OFF",
```

Comment:
Could you also add "1": "ON" here. Could be that some pumps do return that value. If we don't have it listed those pumps will break.

Reply:
Good point.